common/build/eslint-config-fluid/strict-ts.json

```diff
@@ -0,0 +1,1440 @@
+{
+  "env": {
+    "es6": true,
+    "browser": true,
+    "node": true
+  },
+  "globals": {
+    "Atomics": "readonly",
+    "SharedArrayBuffer": "readonly"
+  },
+  "parser": "/workspaces/FluidFramework/common/build/eslint-config-fluid/node_modules/@typescript-eslint/parser/dist/index.js",
+  "parserOptions": {
+    "ecmaVersion": "latest",
+    "sourceType": "module",
+    "ecmaFeatures": {
+      "jsx": true
+    },
+    "project": "./tsconfig.json"
+  },
+  "plugins": [
+    "promise",
+    "@typescript-eslint",
+    "@rushstack/security",
+    "@rushstack",
+    "eslint-comments",
+    "import",
+    "react",
+    "tsdoc",
+    "unicorn",
+    "editorconfig"
+  ],
+  "reportUnusedDisableDirectives": true,
+  "rules": {
+    "@typescript-eslint/explicit-member-accessibility": [
+      "error",
+      {
+        "accessibility": "explicit",
+        "overrides": {
+          "accessors": "explicit",
+          "constructors": "no-public",
+          "methods": "explicit",
+          "properties": "explicit",
+          "parameterProperties": "explicit"
+        }
+      }
+    ],
```

Comment:
This requires explicit accessibility for everything *except* constructors, which are public by default.

Reply:
Isn't everything public by default?  https://www.typescriptlang.org/docs/handbook/2/classes.html#public

Reply:
Yes, sorry. What I meant was that people coming from languages other than JS find public by default to be very... unexpected. Requiring explicit declaration makes it much easier to reason about the code.

My rationale for constructors was that in most cases public  constructors are what devs expect. This is true in other languages as well. That said, when in doubt, consistency should lead the way, so let's change the constructors to be explicit too.